backend/dataall/core/environment/api/input_types.py

```diff
@@ -28,13 +28,11 @@
         gql.Argument('description', gql.String),
         gql.Argument('AwsAccountId', gql.NonNullableType(gql.String)),
         gql.Argument('region', gql.NonNullableType(gql.String)),
-        gql.Argument('vpcId', gql.String),
-        gql.Argument('privateSubnetIds', gql.ArrayType(gql.String)),
-        gql.Argument('publicSubnetIds', gql.ArrayType(gql.String)),
         gql.Argument('EnvironmentDefaultIAMRoleArn', gql.String),
         gql.Argument('resourcePrefix', gql.String),
-        gql.Argument('parameters', gql.ArrayType(ModifyEnvironmentParameterInput))
-
+        gql.Argument('parameters', gql.ArrayType(ModifyEnvironmentParameterInput)),
+        gql.Argument('mlStudioVPCId', gql.String),
```

Comment:
It is very minor, but could we rename this fields to make them more generic, like VPCId and SubnetsIds? I am imagining the scenario where we use the VPCs in other modules (e.g. the default for Notebooks or new modules e.g. AWS Batch )

Reply:
I named it specifically for ML Studio in the case there are other networking resource we also reference / import on creation/update of a new environment sometime in the future

That is why here it is `mlstudioVPC` and in SageMakerStudioDomain clas it is just `vpcId`. If you think best to keep it more generic in Environment input types as well I can make that change but let me know your thoughts @dlpzx

Reply:
Okey, let's keep it as mlstudioVPC, but we need to find a way of referencing module input parameters as we do with the modules enablement parameters `mltudiosEnabled` and so on